src/main/java/chess/piece/Team.java

```diff
@@ -1,43 +1,46 @@
 package chess.piece;
 
+import java.util.List;
+
 public enum Team {
-    BLACK, WHITE, EMPTY;
-    
-    private static final int MAX_ROW_NUMBER_OF_BLACK_TEAM = 2;
-    private static final int MIN_ROW_NUMBER_WHITE_TEAM = 7;
-    private static final int MIN_ROW_NUM = 1;
-    private static final int MAX_ROW_NUM = 8;
-    
-    public static Team from(int rowNum) {
+    BLACK,
+    WHITE,
+    EMPTY;
+
+    private static final List<String> BLACK_TEAM_ROW_NUMS = List.of("1", "2");
+    private static final List<String> WHITE_TEAM_ROW_NUMS = List.of("7", "8");
```

Comment:
연관된 항목으로 보이는데 각 인스턴스의 필드로 관리하면 어떨까요?

```suggestion
    BLACK(List.of("1", "2")),
    WHITE(List.of("7", "8")),
    EMPTY(...);
```

이렇게되면 각 요소를 순회함으로써 `ROW_NUMS`도 제거할 수 있을것같네요!